CHANGELOG.md

```diff
@@ -1,3 +1,184 @@
+## 1.36.1 (September 29, 2024)
+* Add support for Azure Org AWP SSE CMK encrypted disks scan by to Azure org by @DanamoCP in ([#226](https://github.com/dome9/terraform-provider-dome9/pull/226))
```

Comment:
Pls remove:
"@DanamoCP in ([#226](https://github.com/dome9/terraform-provider-dome9/pull/226))"